pkg/templates/generic_lookup_func.go

```diff
@@ -143,6 +148,10 @@ func (t *TemplateResolver) getOrList(
 			return nil, apierrors.NewNotFound(scopedGVRObj.GroupResource(), name)
 		}
 
+		if templateResult != nil && kind == "Secret" {
```

Comment:
In the description, "secret or decrypted data in the template", where is "decrypted data"

Reply:
@yiraeChristineKim that's handled in `processEncryptedStrs`.